src/pal/tests/palsuite/c_runtime/pow/test1/test1.cpp

```diff
@@ -106,13 +106,19 @@ int __cdecl main(int argc, char **argv)
         { -2.7182818284590452,     1,                     -2.7182818284590452,        PAL_EPSILON * 10 },     // x: -(e)                       expected: e
         { -2.7182818284590452,     PAL_POSINF,             PAL_POSINF,                0 },                    // x: -(e)
 
+        { -1.0,                    PAL_NEGINF,             1.0,                       PAL_EPSILON * 10 },
```

Comment:
As for why PAL_EPSILON * 10, the comment on PAL_EPSILON explains.

Basically, we are allowed 15-17 digits of precision. That is 0.xxxxxxxxxxxxxxxxx (validate against PAL_EPSILON) or x.xxxxxxxxxxxxxxxx (PAL_EPSILON * 10) or xx.xxxxxxxxxxxxxxx (PAL_EPSILON * 100) or even 0.0xxxxxxxxxxxxxxxxx (PAL_EPSILON / 10).

If we just used PAL_EPSILON everywhere, we would get incorrect results for larger or smaller results.